enterprise/api/v1/views.py

```diff
@@ -76,39 +108,36 @@ def courses(self, request, pk=None):
 
         Only courses with active course runs are returned. A course run is considered active if it is currently
         open for enrollment, or will open in the future.
```

Comment:
@brittneyexline See https://openedx.atlassian.net/browse/ENT-341?focusedCommentId=278887&page=com.atlassian.jira.plugin.system.issuetabpanels%3Acomment-tabpanel#comment-278887 and the comment before that -- as far as I know from @douglashall, this endpoint will make a transition in future stories.